aws/rust-runtime/aws-inlineable/src/http_response_checksum.rs

```diff
@@ -60,9 +61,9 @@ where
         "ResponseChecksumInterceptor"
     }
 
-    fn read_before_serialization(
+    fn modify_before_serialization(
```

Comment:
Do we need `modify_before_serialization` since I'm trying to see where `context` is being mutated after the change...

Reply:
Good catch! I think this was a leftover of when I was trying to set the default value on the input before I realized it needed to be in a codegened interceptor instead of an inlineable one. Fixed in https://github.com/smithy-lang/smithy-rs/pull/3845/commits/0c6059b5a5a5aed884fe927b22c3670ef3cd57d6